pyproject.toml

```
@@ -37,6 +37,7 @@ isort = "^4.3.21"
 mypy = ">=0.780,<0.791"
 pytest-asyncio = ">=0.12,<0.15"
 outcome-utils = "^4.11.0"
+pact-python = "^1.2.7"
```

We saw some issues when pact tries to upgrade to 1.2.8, so I suggest we pin the version at 1.2.7 for now.

```suggestion
pact-python = "1.2.7"
```